app/controllers/declaration/DispatchLocationController.scala

```diff
@@ -42,30 +42,29 @@ class DispatchLocationController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with ModelCacheable {
 
-  def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
+  def displayPage(): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
```

Comment:
I don't mind having the mode on the `request`, but I still think it needs to be here so we can construct the URLs using Play routing. It will be too easy to miss it out otherwise.